src/types.ts

```diff
@@ -1,23 +1,23 @@
 import type { PageRequestSchema, PageResponseSchema } from './lib/types/index.js';
 
 export type {
-  APIKeyRepresentation,
```

Comment:
I'm full :sickos: mode over here at these imports being autofixed to be properly alphabetized now.